.DOMjudge-release.yml

```diff
@@ -0,0 +1,28 @@
+# The jobs to do a manual DOMjudge release
+createtar:
+  stage: testing
+  image: domjudge/gitlabci:2.1
+  script:
+    - echo -e "\e[0Ksection_start:`date +%s`:gpgsetup\r\e[0KSetup of the signing key"
+    - gpg --no-tty --batch --yes --import $sign_private
+    - echo -e "\e[0Ksection_end:`date +%s`:gpgsetup\r\e[0K"
+    - echo -e "\e[0Ksection_start:`date +%s`:buildrelease\r\e[0KBuild of release tarball"
+    - ./make_release.sh "7.3"
```

Comment:
Probably best to put this in a variable again?

Reply:
I see we already have $VERSION. Let’s use it here?

Reply:
Yes, problem is that due to the current state of the 7.3 branch it will not build if its not hardcoded so still need to look into that.